samples/azure-service-bus-netstandard/native-integration/sample.md

```diff
@@ -46,7 +46,7 @@ For a native message to be processed, NServiceBus endpoints using the Azure Serv
 
 snippet: NecessaryHeaders
 
-NOTE: The `NServiceBus.EnclosedMessageTypes` property must contain the the fully-qualified name of the type expected by the NServiceBus endpoint.
+NOTE: The `NServiceBus.EnclosedMessageTypes` property must contain the namespace and name of the type expected by the NServiceBus endpoint. Some serializers might require also require the fully-qualified name including the assembly of the type.
```

Comment:
Namespace + Type is the fully qualified name

https://docs.microsoft.com/en-us/dotnet/api/system.type.fullname?view=netframework-4.7.2

When assembly is needed as well it’s called 

https://docs.microsoft.com/en-us/dotnet/api/system.type.assemblyqualifiedname?view=netframework-4.7.2


Lets use those names and link to the MS doco?

Reply:
assembly qualified types would only be required if the message type hasn't already been loaded for some reason (unlikely unless you do weird stuff with inheritance)

Reply:
@timbussmann Could be, I don't know. Shouldn't scanning already loaded the assembly?

Reply:
@bording isn't this the stuff you investigated recently?

Reply:
This is definitely related to it, yes.

Reply:
> Shouldn't scanning already loaded the assembly?

not necessarily in case you use unobtrusive message conventions

Reply:
@bording @andreasohlund @timbussmann So should it work without the assembly info? As stated, I've seen deserialization not working due to missing assembly info but it is too long ago to remember the context like which major core version and which serializer.

If you guys state that this is not required then I remove that last sentence.

Reply:
It definitely should work, eg. our Heartbeats use that trick for the heartbeat Message 
https://github.com/Particular/NServiceBus.Heartbeat/blob/develop/src/NServiceBus.Heartbeat/ServiceControlBackend.cs#L25

Reply:
@andreasohlund ok, I'll adjust